src/app/component-status/component-status.component.ts

```diff
@@ -20,17 +20,14 @@ import { componentStatusGhostItems } from './component-status-ghost-items';
 })
```

Comment:
Consider moving the logic to a service, as it will be easier to test. I can see that the code-coverage are starting to complain in the check :)